desktop/resources/app/static/js/textile.js

```diff
@@ -100,11 +95,12 @@ let textile = {
 };
 
 function showOnboarding(screen) {
-  let ob = $(".onboarding")
+  /** @namespace $ **/
+  let ob = $(".onboarding");
   if (ob.hasClass("hidden")) {
     ob.removeClass("hidden")
   }
-  $(".onboarding .screen").addClass("hidden")
+  $(".onboarding .screen").addClass("hidden");
```

Comment:
i think your IDE must be set to <6 javascript... or you've got global wide code style saying to use semis... let's not use them since we don't in any other projects.

Reply:
this is why i had previoulsy included the .idea folder (some of it) so we can share editor settings, but seems weird on a fully public repo

Reply:
hmm, yeh not sure what's up there... i'll remove em. mind sharing your .idea folder?

Reply:
done